renderdoc/driver/vulkan/vk_replay.h

```diff
@@ -550,7 +550,7 @@ class VulkanReplay : public IReplayDriver
     VkSurfaceKHR surface;
     VkSwapchainKHR swap;
     uint32_t numImgs;
-    VkImage colimg[8];
+    VkImage colimg[12];
     VkImageMemoryBarrier colBarrier[8];
```

Comment:
You haven't changed the size of colBarrier here so you're indexing out of bounds. At this point these should be replaced with `rdcarray<VkImage>` and `rdcarray<VkImageMemoryBarrier>` and resized properly (there was no reason to before really, it was probably just early bootstrap hardcoded max that never got changed because drivers were usually sane).